### PR TITLE
UF-234: Configurable Workbench Live Reload

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

File: uberfire-js/src/main/java/org/uberfire/client/exporter/EditorJSExporter.java [hunk 1]
```diff
@@ -62,26 +61,47 @@ public static void registerEditor( final Object _obj ) {
...
-                                                                                                WorkbenchEditorActivity.class,
-                                                                                                Activity.class );
-            beanManager.registerBean( beanDef );
-            beanManager.registerBeanTypeAlias( beanDef, WorkbenchEditorActivity.class );
-            beanManager.registerBeanTypeAlias( beanDef, Activity.class );
-
-            Class<? extends ClientResourceType> resourceTypeClass = getResourceTypeClass( beanManager, newNativeEditor );
-            activityBeansCache.addNewEditorActivity( beanManager.lookupBeans( newNativeEditor.getId() ).iterator().next(), resourceTypeClass );
+
+            JSEditorActivity activity = updateExistentActivityIfExists( beanManager, newNativeEditor );
```
Comment (ederign) [inline uberfire-js/src/main/java/org/uberfire/client/exporter/EditorJSExporter.java]: @paulovmr, could you please review this as we've talked in g+ ?

**Comment by paulovmr** [comment]
@ederign Changes done, could you review it again?

**Comment by paulovmr** [comment]
@ederign I am working on your other request, I'll update the PR asap

**Comment by paulovmr** [comment]
@ederign All done (I only had to update the associated PR).

**Comment by ederign** [comment]
thanks @paulovmr ! https://github.com/uberfire/uberfire/commit/baa8a00a1ff90e86a10c0bcb060640c883a3615c

**Pull request closed without merging**